llvm/bindings/ocaml/llvm/llvm.ml

```diff
@@ -700,6 +700,7 @@ external global_copy_all_metadata : llvalue -> (llmdkind * llmetadata) array
 external is_global_constant : llvalue -> bool = "llvm_is_global_constant"
 external set_global_constant : bool -> llvalue -> unit
                              = "llvm_set_global_constant"
+external global_set_metadata : llvalue -> llmdkind -> llmetadata -> unit = "llvm_global_set_metadata"
```

Comment:
Even though there are some lines that go past 80 columns, I would like to format this change as follows to be consistent with most of the file.

```suggestion
external global_set_metadata : llvalue -> llmdkind -> llmetadata -> unit
                             = "llvm_global_set_metadata"
```